modules/rostests/apitests/atl/CAtlList.cpp

```diff
@@ -54,3 +55,67 @@ START_TEST(CAtlList)
     }
     ok(it == NULL, "it does still point to something!\n");
 }
+
+static CStringW
+to_str(const CAtlList<int>& lst)
+{
+    CStringW tmp;
+    POSITION it = lst.GetHeadPosition();
+    while (it != NULL)
+    {
+        int value = lst.GetNext(it);
+        tmp.AppendFormat(L"%d,", value);
+    }
+    return tmp;
+}
+
+#define ok_list(lst, expected)                                                                                         \
+    do                                                                                                                 \
+    {                                                                                                                  \
+        CStringW _value = to_str(lst);                                                                                 \
+        ok(_value == (expected), "Wrong value for '%s', expected: " #expected " got: \"%S\"\n", #lst,                  \
```

Comment:
```suggestion
        ok(_value == (expected), "Wrong value for '%s', expected: " #expected " got: '%S'\n", #lst,                  \
```
would be slightly less cryptic.